README.md

```diff
@@ -68,16 +68,13 @@ $ make show
 
 ### Running the Coordinator locally
 
-To run the Coordinator on your local machine, use the command:
+To run the Coordinator on your local machine, you can use the
+`example-config.toml` file:
 
 ```shell
-$ python xain_fl/cli.py -f test_array.npy
-```
-
-For more information about the CLI and its arguments, run:
-
-```shell
-$ python xain_fl/cli.py --help
+coordinator --config example-config.toml
+# or
+python xain_fl --config example-config.toml
```

Comment:
It's a bit confusing to have two different commands to run app with no explanations which one is better and what's the difference between them. Will it be possible to add one line of explanations?

Reply:
As far as I know these two commands are pretty much equivalent, except that the second one doesn't require `xain_fl` to be installed. I'll just remove the second one maybe?

Reply:
We can leave both of them, the one for development and one for running coordinator locally, would be good to have short explanation of the difference.